testing/tests/DevExpress.ui.widgets.form/form.materialTheme.tests.js

```diff
@@ -0,0 +1,89 @@
+import $ from 'jquery';
+import 'ui/text_area';
+import 'ui/autocomplete';
+import 'ui/calendar';
+import 'ui/date_box';
+import 'ui/drop_down_box';
+import 'ui/html_editor';
+import 'ui/lookup';
+import 'ui/radio_group';
+import 'ui/tag_box';
+
+import 'common.css!';
+import 'material_blue_light.css!';
+
+QUnit.testStart(function() {
+    const markup = '<div id="form"></div>';
+
+    $('#qunit-fixture').html(markup);
+});
+
+QUnit.module('Form -> material theme');
+
+QUnit.test('Form have correct padding (T849353)', function(assert) {
+    const $form = $('#form').dxForm({
+        colCount: 1,
+        items: [ {
+            itemType: 'group',
+            caption: 'System Information',
+            items: [
+                'input1', {
```

Comment:
Better use the 'DataField1, DataField 2 and ....' instead of the 'input1'.

Reply:
Fixed